### PR TITLE
feature/dependency-check

### PR DESCRIPTION
**Pull request by nathanmartinszup**
**- What I did**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: internal/services/formatters/generic/dependency_check/entities/dependence.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package entities
+
+import (
+	"strings"
+)
+
+type Dependence struct {
```
Comment (matheusalcantarazup) [inline internal/services/formatters/generic/dependency_check/entities/dependence.go:7]: **Dependency**

File: internal/services/formatters/generic/dependency_check/entities/vulnerability.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	case Moderate, Medium:
+		return severities.Medium
+	case Low:
+		return severities.Low
+	default:
+		return severities.Unknown
+	}
+}
+
+func (v *Vulnerability) GetDescription() string {
```
Comment (matheusalcantarazup) [inline internal/services/formatters/generic/dependency_check/entities/vulnerability.go:40]: Why use a method if `Description` is a public field?

File: internal/services/formatters/generic/dependency_check/formatter.go [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+		Language: languages.Generic,
+	}
+
+	return analysisData.SetData(f.GetCustomImageByLanguage(languages.Generic), images.Generic)
+}
+
+func (f *Formatter) parseOutput(output string) error {
+	var analysis *dependencyCheckEntities.Analysis
+
+	index := strings.Index(output, "{")
```
Comment (matheusalcantarazup) [inline internal/services/formatters/generic/dependency_check/formatter.go:77]: The output is not a entire json? Why we  need to find the index of `{`?

File: internal/services/formatters/generic/dependency_check/entities/analysis.go [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package entities
```
Comment (matheusalcantarazup) [inline internal/services/formatters/generic/dependency_check/entities/analysis.go:1]: Missing Copyright. Others files are missing copyright too.

File: internal/services/formatters/generic/dependency_check/formatter.go (hunk 3 above)
Comment (nathanmartinszup) [reply internal/services/formatters/generic/dependency_check/formatter.go:77]: Json comes with a text at the beginning, the index is used to remove this text.

**Review by wiliansilvazup — APPROVED** [review]

**Pull request merged**